tests/test_addDistortionModel.py

```diff
@@ -0,0 +1,124 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import lsst.utils.tests
+import lsst.afw.geom as afwGeom
+from lsst.afw.coord import IcrsCoord
+from lsst.afw.image import ExposureF, ExposureInfo, Calib, VisitInfo
+from lsst.afw.geom import makeCdMatrix, makeSkyWcs
```

Comment:
I just seems silly to me to have these individual imports in addition to afwGeom.  I will again state my preference for the `import ... as ...` style so that it is clear where the functions came from when they are used lower down in a file without having to look back up to the top (in addition to being consistent with a lot of existing code).